### PR TITLE
Fixes #1064 - made search bar distinct from the background

### LINKED ISSUE #1064: Search bar on the home page doesn't seem have a distinction from the background.
After clicking the* Search button* on the Webcompat.com home page, the
search box doesn't seem have a distinction from the background.

May be improving the UI text box around it would make the look and feel better.

2016-05-24_18h46_39

2016-05-24_18h59_28

Suggestion

search bar

### PR DESCRIPTION
**Pull request by deepthivenkat**
r? @miketaylr

### CODE REVIEW

**Comment by miketaylr** [comment]
@deepthivenkat for CSS stuff, I recommend we ask @magsout for review -- he's sort of an expert. :)

Here's a gif of the change (useful when making visual/UI PRs)
search

File: webcompat/static/css/development/components/search-bar.css [hunk 1]
```diff
@@ -39,7 +39,12 @@
       font-size: var(--SearchBar-fontSize);
       flex: 1;
       margin: 0 1em;
-      background-color: transparent;
+      background-color: white;
+      padding-top: 5px;
```
Comment (miketaylr) [inline webcompat/static/css/development/components/search-bar.css]: You can use shorthand padding here. Not sure if we want to use px or em, will let Guillaume comment there.

File: webcompat/static/css/development/components/search-bar.css [hunk 2]
```diff
@@ -39,7 +39,12 @@
       font-size: var(--SearchBar-fontSize);
       flex: 1;
       margin: 0 1em;
-      background-color: transparent;
+      background-color: white;
```
Comment (miketaylr) [inline webcompat/static/css/development/components/search-bar.css]: We have a variable `--base-background` -- see style-variable.css

File: webcompat/static/css/development/components/search-bar.css (hunk 1 above)
Comment (magsout) [inline webcompat/static/css/development/components/search-bar.css]: agree with @miketaylr 

and em is better: padding: .31em

>

File: webcompat/static/css/development/components/search-bar.css (hunk 2 above)
Comment (magsout) [inline webcompat/static/css/development/components/search-bar.css]: Same here

Comment (magsout) [inline webcompat/static/css/development/components/search-bar.css]: add new var for this background

Comment (miketaylr) [inline webcompat/static/css/development/components/search-bar.css]: @magsout something like `--SearchBar-background`? I suppose that could just inherit from `--base-background`?

Comment (miketaylr) [inline webcompat/static/css/development/components/search-bar.css]: @deepthivenkat you can add a new variable in this section: https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/css/development/components/search-bar.css#L4

Comment (magsout) [inline webcompat/static/css/development/components/search-bar.css]: Yes exactly. Sorry for my previous simple without explanation  comment

**Comment by miketaylr** [comment]
Looks good, thanks!

**Pull request merged**